packages/cli/src/events/log-event-relay.ts

```diff
@@ -1,72 +1,60 @@
 import { Service } from 'typedi';
-import { MessageEventBus } from './MessageEventBus/MessageEventBus';
+import { MessageEventBus } from '@/eventbus/MessageEventBus/MessageEventBus';
 import { Redactable } from '@/decorators/Redactable';
-import { EventService } from './event.service';
-import type { Event } from './event.types';
+import { EventRelay } from '@/events/event-relay';
+import type { RelayEventMap } from '@/events/relay-event-map';
 import type { IWorkflowBase } from 'n8n-workflow';
+import { EventService } from './event.service';
 
 @Service()
-export class AuditEventRelay {
+export class LogEventRelay extends EventRelay {
```

Comment:
Could we name this `LogStreamingEventRelay` to make it match the name of the feature it's related to ([Log streaming](https://docs.n8n.io/log-streaming/))? `LogEventRelay` might make me think it's about logging to console log